science/ncbi_tools/Portfile

```diff
@@ -1,52 +1,79 @@
 PortSystem 1.0
 
 name            ncbi_tools
-version         20080302
+version         20120620
 description     blast is a set of tools for doing nucleotide and protein searches
 maintainers     nomaintainer
+license         public-domain
 categories      science
 platforms       darwin
 homepage        https://www.ncbi.nlm.nih.gov/blast/
 fetch.use_epsv  no
-master_sites    ftp://ftp.ncbi.nlm.nih.gov/toolbox/ncbi_tools/old/20080302
+master_sites    ftp://ftp.ncbi.nlm.nih.gov/toolbox/ncbi_tools/old/${version}
 distfiles       ncbi.tar.gz
-checksums       md5 f8de9d7264aed7de1d87d2185df2e07c
+checksums       md5 e84826b8d390ca1ec6d0dfcaaf4cb6aa \
```

Comment:
The `md5` checksum is no longer needed.  You can drop that one, and keep `sha256`, `rmd160` and `size`.

Reply:
But it's fine to keep md5 if it's what upstream publishes. (Yes, of course they should use a better hash function.)

Reply:
Fixed in the 2nd "proposed cleanup" commit